packages/extension-horizontal-rule/src/horizontal-rule.ts

```diff
@@ -9,7 +9,13 @@ export interface HorizontalRuleOptions {
    * @default {}
    * @example { class: 'foo' }
    */
-  HTMLAttributes: Record<string, any>
+  HTMLAttributes: Record<string, any>,
+  /**
+   * The default type to insert after the horizontal rule.
+   * @default "paragraph"
+   * @example "heading"
+   */
+  defaultType: string,
```

Comment:
I like the change, but I feel like defaultType could be more descriptive.

Maybe something like:
 - insertAfter
 - afterType